src/pages/iou/IOURequestPage.js

```diff
@@ -1,7 +1,12 @@
 import React from 'react';
+import ScreenWrapper from '../../components/ScreenWrapper';
 import IOUModal from './IOUModal';
 
 export default props => (
     // eslint-disable-next-line react/jsx-props-no-spreading
-    <IOUModal {...props} />
-);
+    <ScreenWrapper>
```

Comment:
What does this do?

Reply:
`ScreenWrapper` is used in the Modal views to provide consistent padding and other styles. The important fix for us is that the status bar for mobile will no longer overlap the Modal header.